samples/sensor/sensor_shell/sample.yaml

```diff
@@ -19,6 +19,8 @@ tests:
     filter: CONFIG_SERIAL and dt_chosen_enabled("zephyr,shell-uart")
     min_ram: 40
     harness: pytest
+    timeout: 180
+    slow: true
```

Comment:
this will disable the test completely right now (we do not run slow tests in CI), I would stick to the change in timeout only.

Reply:
I haven't known about that. thanks for information and review.
Change is applied.